Rewrote the "How to Launch Server on...." sections

MoltonMontro opened the pull request:
Rewrote the "How to Launch Server on..." sections for Windows and Linux. Includes new images to help visually show steps, and includes a new set of instructions for viewing file name extensions.

SDGNelson commented:
Thanks for the improvements. There are a few edits I would like to propose however:
1. The tone is different than the other instructional documentation. For audio/video tutorials I think "we" works well, but for text tutorials not so much. Let me know if you think this should change though.
2. Right now the images are hosted on GitHub, but I would prefer that you embed them in the repo. This way they are visible in an offline copy.
3. Typo 2.i probably means if the extension is NOT visible?
4. File path under Linux 1. should use forward slashes, not Windows backslashes.
5. This is more personal preference, but I think we should avoid non-standard markdown like the superscript tag. Maybe not a big deal though.

MoltonMontro commented:
1. I am inclined to disagree on the basis of readability and friendliness, however, as long as it doesn't detract from what readers need to know. However, my original intent was not to include "we" and the other informal/casual tones used as to match preexisting documentation, so I've removed them in these latest commits.
2. Added images to repo, and linked to images from ServerHosting.md. Images also have a thinner outline.
3. Fixed typo.
4. Fixed typos.
5. It shouldn't be an issue and I feel it's safe to assume users won't run into issues with sub/sup tags specifically. Most markdown implementations will be compatible with superscript and subscript. However, I've removed the superscript due to formatting concerns instead (more specifically, me having used the semantic superscript for better readability in place of the lack of styling/CSS).

SDGNelson commented:
Thanks for the adjustments! Okay, I will merge this now.
